jax/core.py

```diff
@@ -280,7 +306,7 @@ def read(v):
   def write(v, val):
     env[v] = val
 
-  env = {}
+  env: Dict[Var, Any] = {}
```

Comment:
I think this should be `Dict[Union[Literal, Var], Any]` or maybe `Dict[Atom, Any]` where we define `Atom = Union[Literal, Var]`.

Reply:
Whoops nevermind, got that wrong.